api/src/main/java/ca/bc/gov/educ/api/trax/service/institute/SchoolService.java

```diff
@@ -21,18 +28,20 @@ public class SchoolService {
 
 	@Autowired
 	private EducGradTraxApiConstants constants;
-
 	@Autowired
 	private WebClient webClient;
-
 	@Autowired
 	SchoolRedisRepository schoolRedisRepository;
-
+	@Autowired
+	SchoolDetailRedisRepository schoolDetailRedisRepository;
 	@Autowired
 	SchoolTransformer schoolTransformer;
-
+	@Autowired
+	SchoolDetailTransformer schoolDetailTransformer;
 	@Autowired
 	private RestUtils restUtils;
+	@Autowired
+	RedisTemplate<String, String> redisTemplate;
 
 	public List<School> getSchoolsFromInstituteApi() {
 		try {
```

Comment:
Create a webclient and qualifier for grad-edx-client like the pattern established [here](https://github.com/bcgov/EDUC-GRAD-TRAX-API/blob/main/api/src/main/java/ca/bc/gov/educ/api/trax/config/GradTraxConfig.java#L56) and used [here](https://github.com/bcgov/EDUC-GRAD-TRAX-API/blob/main/api/src/main/java/ca/bc/gov/educ/api/trax/util/CommonSchoolCache.java#L29) and configured [here](https://github.com/bcgov/EDUC-GRAD-TRAX-API/blob/main/api/src/main/resources/application.yaml#L47). Also we should add some resilience to the call in case of a 5xx response (like 503 due to pod restarts, etc.) There is an example of how you can add simple resilience to REST calls [here](https://github.com/bcgov/EDUC-GRAD-GRADUATION-API/blob/main/api/src/main/java/ca/bc/gov/educ/api/graduation/service/RESTService.java#L73)